.github/workflows/task-test.yml

```diff
@@ -427,6 +427,17 @@ jobs:
           CLEAR_LOGS: 0
           ENABLE_ASSERT: 1
         run: make unit-tests
+      - name: Rust tests (MIRI)
+        if: inputs.san == 'address'
+        timeout-minutes: ${{ fromJSON(inputs.test-timeout) }}
+        id: rust_unit_tests_miri
```

Comment:
We should add this to the conditional checks in later steps to determine failure/success.